src/calculateRentalCost.js

```diff
@@ -28,7 +28,15 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let result = days * 40;
+
+  if (days > 2 && days < 7) {
+    result = result - 20;
+  } else if (days > 6) {
+    result = result - 50;
+  }
```

Comment:
Check checklist)
20, 40, 50 - magic numbers
You should create variables for this values